cmd/ext/create.go

```diff
@@ -226,9 +227,14 @@ func getWorkspaceProjects() []string {
 	e := filepath.Walk(
 		workspaceDir,
 		func(path string, info os.FileInfo, err error) error {
-			if err == nil && info.Name() == "client-extension.yaml" {
-				fullPathDir := filepath.Dir(path)
-				projectSet[fullPathDir[len(workspaceDir):]] = novalue
+			if err == nil {
+				if strings.HasPrefix(path, filepath.Join(workspaceDir, ".git")) {
```

Comment:
Maybe we should ignore all .gitignore folders too?  So we can avoid things like node_modules?

Reply:
hmm, perhaps that would be more performant (not that it took more than a few millis but sure)

Reply:
I fear building a .gitignore parser might not be worth the effort here. I'll look to see if there's a go library which does this.

Reply:
skipping this as discussed